frontend/packages/console-plugin-sdk/src/codegen/active-plugins.ts

```diff
@@ -13,8 +15,44 @@ import { Extension, ActivePlugin } from '../typings';
 import { trimStartMultiLine } from '../utils/string';
 import { consolePkgScope, PluginPackage } from './plugin-resolver';
 
+const getExtensionsFilePath = (pkg: PluginPackage) => path.resolve(pkg._path, extensionsFile);
+
+const guessFileExtension = (basePath: string) => {
+  const extensions = ['.tsx', '.ts', '.js', '.jsx'];
+
+  if (fs.existsSync(basePath)) {
+    return basePath;
+  }
+
+  for (const ext of extensions) {
+    if (fs.existsSync(`${basePath}${ext}`)) {
+      return `${basePath}${ext}`;
+    }
+  }
+
+  return basePath;
+};
+
+const getExposedModuleFilePath = (pkg: PluginPackage, moduleName: string) => {
+  const modulePath = path.resolve(pkg._path, pkg.consolePlugin.exposedModules[moduleName]);
```

Comment:
Does this include index files? Without `guessModuleFilePath` many static plugins produce warnings, e.g., 

```
WARNING in ./get-active-plugins.js
Module Warning (from ./node_modules/val-loader/dist/cjs.js):
Exposed module 'workload' in static plugin @console/topology refers to non-existent file ./frontend/packages/topology/src/components/workload
```

since the corresponding exposed module points to a folder and not a file, e.g.,

https://github.com/openshift/console/blob/26d0f58cddd8d36e72acd3e1e7d6c10fb6982d49/frontend/packages/topology/package.json#L22